engine/src/test/suite/cfg-tests.xml

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Hibernate Validator Configuration Tests" verbose="1">
```

Comment:
I think it was just used to run the tests he was initially working on. I'll remove it.